### ISSUE TITLE
Syncronized charts?

### ISSUE THREAD

**Issue by skuske** [highcharts/highcharts-ios#96]
Is this possible with the iOS wrapper?

https://www.highcharts.com/demo/synchronized-charts

**Comment by ihnatmoisieiev** [comment]
Hi @skuske,
it is not possible with the iOS wrapper.
You can create charts separately.

**Comment by skuske** [comment]
Hello @ihnatmoisieiev 

> You can create charts separately.

I know that, but how can I sync touch events between multiple charts?

**Comment by ihnatmoisieiev** [comment]
@skuske unfortunately, it isn't possible.

**Comment by arunavasumathi** [comment]
@skuske did you achieve synchronised charts?

**Comment by skuske** [comment]
@arunavasumathi 

No, as @ihnatmoisieiev mentioned above: it is not possible with the iOS wrapper.

**Comment by arunavasumathi** [comment]
@skuske thank you.

@ihnatmoisieiev - please let me know if this feature will be given in the future or not. Any other work around?

**Comment by ihnatmoisieiev** [comment]
@arunavasumathi as I write above, unfortunately, it isn't possible. 
I suppose you can try to simulate it by yourself with feature that will be added in the next release.

**Comment by arunavasumathi** [comment]
sure i will try

**Comment by arunavasumathi** [comment]
@ihnatmoisieiev i tried simulating it, but needs little fine tuning. Could you help me achieve it if i share the code

**Comment by ihnatmoisieiev** [comment]
@arunavasumathi unfortunately, we have never tried to implement it. So, I don't know if I can help you, but what do you exactly need?

**Comment by arunavasumathi** [comment]
@ihnatmoisieiev ok, please help me on the following:
1. The marker at the mouseOut point remains selected at times and unselected at times in which case the start of the mouseOver points remains selected
![MouseMove11](https://user-images.githubusercontent.com/30022843/55058033-dfb1e080-5090-11e9-8e86-ff023e235398.gif)


2. The marker sometimes becomes point selected, even if I don't click on the point like below
![MouseMove12](https://user-images.githubusercontent.com/30022843/55057975-bc873100-5090-11e9-88e9-ba2739a34532.gif)


Here is my code:

        let anim = HIAnimationOptionsObject()
        anim.duration = 0
        
        let pointSelect = "function() { var currentPoint = this; this.series.data.forEach(function (point) { if (point !== currentPoint) { point.setState(''); } }); }"
        
        let chartFunction = HIFunction(closure: { (context) in
            
            let xIndex = context!.getProperty("this.x") as! Int
            
            self.pointToSelect = xIndex
            self.delegate1?.PrintNumbers(index: xIndex)
            self.delegate2?.fnReload(chartType: 2, atIndex: xIndex)
        }, jsFunction: pointSelect, properties: ["this.x"])
        
        let jsFun = "function () { this.series.forEach(function(serie) { serie.data[" + "\(pointToSelect)" + "].setState('hover'); }); }"
        
        
        let plotOptions = HIPlotOptions()
        plotOptions.series = HISeries()
        plotOptions.series.animation = anim
        plotOptions.series.allowPointSelect = true
        plotOptions.series.point = HIPoint()
        plotOptions.series.point.events = HIEvents()
        plotOptions.series.point.events.click = chartFunction
        plotOptions.series.point.events.mouseOver = chartFunction
        
        let chartFunction1: HIClosure = { context in
            let xValue = context!.getProperty("event.xAxis[0].value") as! Double
            var intValue = Int(xValue)
            let d : Double = xValue - Double(intValue)
            if d > 0.5 {
                intValue += 1
            }
            
            self.pointToSelect = intValue
            self.delegate1?.PrintNumbers(index: intValue)
            self.delegate2?.fnReload(chartType: 2, atIndex: intValue)
        }
        
        let chartLoadFn: HIClosure = {context in
            print("Load function delegate call inside HLine")
        }
        
        chartLine.zoomType = "xy"
        chartLine.events = HIEvents()
        chartLine.events.click = HIFunction(closure: chartFunction1, properties: ["event.xAxis[0].value"])
        chartLine.events.load = HIFunction(closure: chartLoadFn, jsFunction: jsFun)
        chartLine.events.mouseOver = HIFunction(closure: chartFunction1, jsFunction: jsFun, properties: ["event.xAxis[0].value"])

**Comment by arunavasumathi** [comment]
Also with bar charts for a similar data set and same chart options, the behaviour is like this
![MouseMove2](https://user-images.githubusercontent.com/30022843/55098058-20d2e080-50e3-11e9-8a73-e4f3c88c1c1b.gif)

**Comment by AlexanderMatveev** [comment]
@arunavasumathi I think your last question is a little off topic and you should create new Issue
